tools/intersmash-tools-provisioners/pom.xml

```diff
@@ -233,6 +233,17 @@
                             <targetPackage>org.jboss.intersmash.model.helm.charts.values.wildfly</targetPackage>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>charts.values.generate-schema-model.eap8</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <sourceDirectory>${basedir}/src/main/resources/org/jboss/intersmash/tools/provision/helm/values/schema/eap8</sourceDirectory>
```

Comment:
personally I'd appreciate some comment reminding that file `eap8-helm-values.yaml` is actually from https://github.com/jbossas/eap-charts/blob/eap8/charts/eap8/values.schema.json

Reply:
+1, fixing.